### PR TITLE
Multiple widget theming fix!

### PR DESCRIPTION
**Pull request by mgord9518**
Less than ideal, but it *is* functional. Do *NOT* style any widgets
besides the first 18. If this somehow really is the best way to solve
this issue, I'll add the rest

### CODE REVIEW

File: fltk.cxx [hunk 1]
```diff
@@ -32,6 +31,16 @@ void go_fltk_init_styles(void) {
     Fl::use_high_res_GL(1);
 }
 
+typedef void (*f_array)(int, int, int, int, unsigned int);
+
+// idk if it's worth adding all the boxtypes this way, but this is enough to
+// override all the default designs
+f_array currentBoxTypeCb[56] = {&_go_drawBox0,  &_go_drawBox1,  &_go_drawBox2,  &_go_drawBox3,
```
Comment (pwiecz) [inline fltk.cxx:38]: Why 56, when there are only 19 elements?

File: fltk.go [hunk 2]
```diff
@@ -50,16 +50,11 @@ func SetBoxType(b BoxType, d func(int, int, int, int, uint), o ...int) {
 		o = append(o, []int{0, 0, 0, 0, 0}...)
 	}
 
-	setBoxTypeCb = d
+	setBoxTypeCb[b] = d
```
Comment (pwiecz) [inline fltk.go:53]: Can you add some more meaningful error message when b > 18?

**Comment by mgord9518** [comment]
It was because that's the total number of boxtypes (but it should've been 57 because I didn't account for box 0). I ended up just adding the rest, now if someone tries to create a box > 56, it'll panic, saying that defining new BoxTypes isn't yet implemented

**Comment by pwiecz** [comment]
I hope we'll be able to make it beautiful at some in time. ;)

**Pull request merged**